.github/workflows/scripts/release-against-charts.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# Bumps Webhook version in a locally checked out rancher/charts repository
+#
+# Usage:
+#   ./release-against-charts.sh <path to charts repo> <prev webhook release> <new webhook release>
+#
+# Example:
+# ./release-against-charts.sh "${GITHUB_WORKSPACE}" "0.5.0-rc.13" "0.5.0-rc.14"
+
+CHARTS_DIR=$1
+PREV_WEBHOOK_VERSION=$2   # e.g. 0.5.2-rc.3
+NEW_WEBHOOK_VERSION=$3
+
+usage() {
+    cat <<EOF
+Usage:
+  $0 <path to charts repo> <prev webhook release> <new webhook release>
+EOF
+}
+
+if [ -z "$CHARTS_DIR" ] || [ -z "$PREV_WEBHOOK_VERSION" ] || [ -z "$NEW_WEBHOOK_VERSION" ]; then
+    usage
+    exit 1
+fi
+
```

Comment:
We may want to check that `$PREV_WEBHOOK_VERSION != $NEW_WEBHOOK_VERSION`

Reply:
Addressed in a7c70c3299b2f955c33ce7b01873c1e63a0d5df4.